Makefile

```diff
@@ -1,4 +1,4 @@
-DEVVM=work
+DEVVM=securedrop-dev
```

Comment:
I've been using `sd-dev`, which means I need to edit the Makefile every time I run the `make clone` action from dom0. Fine with that workflow for now, just noting the target VM change in case it affects others.

Reply:
hmm, probably should keep it as `work` (e.g. we have comments in a few places in the README referencing the example `work` AppVM)

Reply:
Oops, that should not have snuck in there- I'll move it back to `work`.

I've taken to editing `DEVVM` on my development VM so `make clone` works in dom0 consistently. I wonder if a different workflow would make more sense- setting an environment var maybe?

Reply:
The env var approach (for both `DEVVM` and `DEVDIR`) would satisfy my needs; we could port to a small script in `scripts/` and preserve the defaults there via e.g. `devvm="${DEVVM:-work}"`. 

Zooming out a bit, the whole take-a-lap-through-dom0 wouldn't be necessary if we configured the admin API to let the dev VM, whatever it is, create other VMs—but that's best discussed in a separate issue.